feelpp/feel/feelcore/environment.hpp

```diff
@@ -656,8 +656,8 @@ class FEELPP_EXPORT Environment
         ( required
           ( name,( std::string ) ) )
         ( optional
-          ( sub,( std::string ),"" )
```

Comment:
we never had any issue with this. I don't see why we cannot do this, there is a constructor with `const char*`
https://en.cppreference.com/w/cpp/string/basic_string/basic_string

what is the error message exactly ?

Reply:
How do you take the `.size()` of a `const char*`?
```
feelpp/feelpp/feel/../feel/feelcore/environment.hpp:669:18: error: member reference base type
      'char const[1]' is not a structure or union
        if ( !sub.empty() )
              ~~~^~~~~~
feelpp/feelpp/feel/../feel/feelcore/environment.hpp:654:5: note: in instantiation of function
      template specialization
```